freeipa/src/main/java/com/sequenceiq/freeipa/kerberosmgmt/v1/KerberosMgmtV1Service.java

```diff
@@ -115,6 +120,13 @@ public void deleteServicePrincipal(ServicePrincipalRequest request, String accou
         String canonicalPrincipal = constructCanonicalPrincipal(request.getServiceName(), request.getServerHostName(), realm);
         ipaClient = freeIpaClientFactory.getFreeIpaClientForStack(freeIpaStack);
         ipaClient.deleteService(canonicalPrincipal);
+        //Delete the vault secrets associated with the service principal
+        if (Strings.isNullOrEmpty(request.getClusterCrn())) {
+            LOGGER.error("Cluster CRN not provided. Auto-generating one");
```

Comment:
I think this should be a warning rather than an error. Same with the other similar error log statements.